packages/clerk-js/src/ui/elements/Avatar.tsx

```diff
@@ -51,14 +49,14 @@ export const Avatar = (props: AvatarProps) => {
   }
 
   const ImgOrFallback =
-    initials && (!avatarExists || error) ? (
+    !avatarExists || error ? (
       <InitialsAvatarFallback {...props} />
     ) : (
       <Image
         elementDescriptor={imageElementDescriptor || descriptors.avatarImage}
         alt={fullName}
         title={fullName}
-        src={src || GRAVATAR_DEFAULT_AVATAR}
+        src={src || ''}
```

Comment:
❓ Don't we need a default fallback in case something goes extremely wrong?

Reply:
@nikosdouvlis in theory we will never reach this state. @SokratisVidros should we keep this gravatar default for such an extreme case?

Reply:
Let's remove it.